plugins/versionlock.py

```diff
@@ -57,6 +58,9 @@ def config(self):
                        and cp.get('main', 'locklist'))
 
     def sack(self):
+        if not self.cli.demands.resolving:
```

Comment:
@jsilhan Thanks it works, but I am not sure if message "Ignoring versionlock for non-transactional operation" cannot be improved. What about something like: "Excludes from Versionlock plugin were not applied". I am also not sure if the message should be only for debug, but in some cases it is better to see it every time.